.changelog/10424.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+connect/ca: cease including the common name field in generated certs
```

Comment:
Might be worth clarifying somehow that generated CA certs still require/include a common name? Not totally happy with the wording, but maybe something like:
```suggestion
connect/ca: cease including the common name field in generated x509 certificates when not used as a CA
```